docs/facility-registry/facility-v2-api.yaml

```
@@ -0,0 +1,382 @@
+            schema:
+              $ref: '#/components/schemas/FacilityCreateRequest'
+      responses:
+        '201':
+          description: Facility created successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Facility'
+        '400':
+          description: Invalid input data
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)